_posts/2016-11-15-hospitalrun-release-1.0.markdown

```diff
@@ -0,0 +1,66 @@
+---
+title: Coming Soon! HospitalRun 1.0
+subtitle: ...and it's about time.
+date: 2016-11-15 00:00:00 Z
+categories:
+- hospitalrun
+- update
+- releases
+layout: post
+---
+
+So by now you're probably wondering, when are we ever going to ship HospitalRun 1.0? 
+
+Well, we finally have some good news to share. After months of iterative requirements work with doctors, we have a GitHub project that defines [what we mean by "done" for release 1.0](https://github.com/HospitalRun/hospitalrun-frontend/projects/2). 
+
+At our current pace, we're confident that HospitalRun 1.0 will be released by or before April 1, 2017; but with your help, it could be a lot sooner!
+
+Of course, the project development will be ongoing; but we want to get HospitalRun to a state where the project team could feel good about facilities downloading and trying the system. Our hope is that those initial deployments will not only drive more feature development but also the creation of one of our long-term goals: an online service for hosted HospitalRun instances.
+
+
+
+# The Feature Set for 1.0
+So what precisely is in HospitalRun 1.0? We're glad you asked. Here's the high-level list.
+<ul>
+<li>Electronic Medical Records, focused primarily around outpatient department (OPD) support</li>
+<li>Queueing systems for Imaging and Labs but without LIS or DICOM integration (though you can work on it :))</li>
+<li>Pre and post operative notes and report</li>
+<li>Discharge notes and report</li>
+<li>Quality Incident Management</li>
+<li>Inventory Management</li>
+<li>pre-release Billing</li>
+<li>Release 1.0 Deployment instructions</li>
+</ul>
+
+# How can you help?
+If you're a development and either have contributed to HospitalRun or have considered it, now would be a great to help us get 1.0 across the proverbial finish line. You can find the project board [here](https://github.com/HospitalRun/hospitalrun-frontend/projects/2). 
+
+To start, [join the project Slack](https://hospitalrun-slackin.herokuapp.com).
```

Comment:
We should probably include the contributing guide somewhere in this paragraph, if not actually instead of the Slack link.